### ISSUE TITLE
clay-bindgen: clang can't find stddef.h

### ISSUE THREAD

**Issue by galchinsky** [jckarter/clay#419]
```
clay/lib-clay/libc/generated$ make linux32
clay-bindgen -builtins -D_FILE_OFFSET_BITS=64 generated_from_unix.h -o generated.linux.32.clay
/usr/include/stdio.h:34:11: fatal error: 'stddef.h' file not found
/usr/include/stdio.h:34:11: fatal error: 'stddef.h' file not found
make: *** [linux32] Ошибка 1
```

A workaround is to add something like `-I/usr/include/clang/3.0/include/` in a Makefile

I'm on debian stable.

Here also this problem: http://static.opengameart.org/logs/clay/%23clay.2012-01-23.log , search by stddef.h

**Comment by stepancheg** [comment]
Have same problem on Ubuntu 12.04.

**Comment by galchinsky** [comment]
fixed in #443

**Comment by jckarter** [comment]
According to @agemogolk LLVM 3.2 fixes this upstream.
